draft-ietf-quic-transport.md

```diff
@@ -3297,9 +3300,10 @@ value of fields.
 
 ## Packet Number Encoding and Decoding {#packet-encoding}
 
-Packet numbers in long and short packet headers are encoded in 1 to 4 bytes.
-The number of bits required to represent the packet number is reduced by
-including the least significant bits of the packet number.
+Packet numbers are integers in the range 0 to 2^62-1 ({{packet-numbers}}).  When
+present in long or short packet headers, they are encoded in 1 to 4 bytes.  The
```

Comment:
do we have to say here that they are big endian encoded?

Reply:
We say that at the top, [here](https://quicwg.org/base-drafts/draft-ietf-quic-transport.html#packet-formats).